NomaiGrandPrix/SpeedrunHarmonyPatches.cs

```diff
@@ -42,25 +43,43 @@ private static void EnableTitleScreen(TitleScreenManager manager, bool shouldEna
   [HarmonyPatch(typeof(TitleScreenManager), nameof(TitleScreenManager.Start))]
   public static void TitleScreenManager_Start_Postfix()
   {
-    SpeedrunState.IsGameStarted = false;
-    SpeedrunState.ModEnabled = false;
+    NomaiGrandPrix.NomaiGrandPrix.Instance.SpeedrunState.IsGameStarted = false;
```

Comment:
We should probably change the name of the namespace so that these can be shorter. I believe the class name itself needs to match exactly though for the mod loader to work.

Reply:
I support that change, but don't want to block this PR on it. I'll create an issue to track the follow-up work.

Reply:
Tracking in #105